### DISCUSSION TITLE
Unable to run

### DISCUSSION THREAD

**Discussion by mmaslar** [section83/MacYTDL#58] [General]
I've successfully been able to use MacYTDL in the past; however, I'm now getting a vague error when trying to open the app.

"The command terminated due to receipt of a signal. (1009).

Has anyone experienced this error and found a fix?

**Comment by section83** [comment]
Hello mmaslar,

Sorry you've had this problem.  That error usually occurs when a dialog is canceled and the code has nowhere else to go – I get that error a lot in testing.  But, it should not occur in the published app.

Can you provide more detail on the error ?

- When does the error occur e.g. when you start MacYTDL or at another time ?
- Which version of macOS are you using ?
- Which version of MacYTDL are you using ?

Thanks.

**Comment by mmaslar** [comment]
Thanks for the reply. I’m not sure what was causing the error, but it was occurring whenever I tried to start MacYTDL. I was using macOS 12.6 and MacYTDL v1.21.2. I’ve since updated to macOS 13.0 and the error no longer appears when I start the app. I’m assuming there was something going on with macOS 12.6 and the update cured it.


Michael Maslar
***@***.***
770.331.8992

**Comment by section83** [comment]
Very odd.  I can't think of what the cause of the problem in 12.6 might have been.  Perhaps a _Full Disk Access_ issue.  Anyway, glad it works in Ventura.

By the way, Ventura has revealed a minor bug in MacYTDL: hiding doesn't always work properly – instead MacYTDL sometimes becomes inactive but modal. No idea on why this is happening as yet but it doesn't affect operation.
